packages/common/src/utils/faker/fakeCertificateData.ts

```diff
@@ -20,6 +20,7 @@ import {
   ConfigUeDropdown,
```

Comment:
Here there are also changes in the master branch, regarding config, and a new file fakeCertificateConfig, that needs to be implementer